### ISSUE TITLE
Enhancement: git extention should include amend commit option

### ISSUE THREAD

**Issue by yakuzaaaa** [Microsoft/vscode#28690]
Like: 
git commit --amend

**Comment by ramya-rao-a** [comment]
@yakuzaaaa Are you referring to the out of the box git behavior or any particular git extension?

**Comment by Gruntfuggly** [comment]
I would guess the out of the box behaviour, because it seems to be missing? I can see there is an "undo last commit", but this unstages everything from the last commit, which is not the same thing as amending the current commit. 

git-gui and atom have a simple checkbox which then amends the current commit instead of creating a new one - would it be possible to have the same in vscode? It's the one thing that stops me using vscode's built in git interface.

In other words, you should be able to check and then uncheck the checkbox and be left in the original state. With the "undo last commit", this isn't possible.

**Comment by yakuzaaaa** [comment]
Yes, out of the box

**Comment by Krzysztof-Cieslak** [comment]
`Git commit amend ` command was added some time ago by https://github.com/Microsoft/vscode/pull/17755

**Comment by Gruntfuggly** [comment]
Should there be something in the menu? Please can you tell me where the option is?

**Comment by Krzysztof-Cieslak** [comment]
Hmmm.... looks like it has disappeared for some reason, probably was omitted when git integration was ported to new API... @joaomoreno will probably know if it was by-design, or just someone missed one command.

**Comment by joaomoreno** [comment]
I believe we simply missed it... sorry for that.

**Comment by Krzysztof-Cieslak** [comment]
@joaomoreno: I'll add it again.
